mousestyles/behavior/metrics.py

```diff
@@ -0,0 +1,153 @@
+
+    Returns
+    -------
+    The total amount of time in seconds of the specified mouse-day.
+    """
+    start_seconds, end_seconds = data.load_start_time_end_time(
+        strain, mouse, day)
+    return end_seconds - start_seconds
+
+
+def active_time(strain, mouse, day):
+    """Returns the total amount of active time recorded for a certain mouse-day.
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+
+    Returns
+    -------
+    The total amount of active time in seconds for the specified mouse-day.
+    """
+    return create_intervals('AS', strain, mouse, day).measure()
+
+
+def total_amount(strain, mouse, day, feature):
+    """Returns the total amount consumed/moved for one of the following features:
+     food ("F"), water ("W"), and locomotion ("L").
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+    feature : {'F', 'W', 'L'}
+        The feature used. 'F' for food, 'W' for water, 'L' for locomotion.
+
+    Returns
+    -------
+    The total amount consumed/moved for the given feature in the mouse-day.
+    """
+    df = data.load_all_features()
+    strain_cond = df['strain'] == strain
+    mouse_cond = df['mouse'] == mouse
+    day_cond = df['day'] == day
+
+    df = df[strain_cond & mouse_cond & day_cond]
+
+    if df.shape[0] == 0:
+        raise ValueError("No consumption data available for specified mouse-day.")
+
+    if feature is 'W':
+        col_name = 'Water'
+    elif feature is 'F':
+        col_name = 'Food'
+    elif feature is 'L':
+        col_name = 'Distance'
+    else:
+        raise ValueError("Feature must be one of {'F', 'W', 'L'}.")
+
+    return df[col_name].sum()
+
+
+def create_intervals(activity_type, strain, mouse, day):
+    r"""Returns an interval object on a certain mouse-strain-day
+    for a given activity type.
+
+    Parameters
+    ----------
+    activity_type : str
+        String specifying activity type {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+    strain : int
+        Integer representing the strain of the mouse
+    mouse : int
+        Integer representing the specific mouse
+    day : int
+        Integer representing the day to produce the metrics for
+
+    Returns
+    -------
+    An intervals object `intervals` for the given activity type and
+    mouse-strain-day.
+
+    Examples
+    --------
+
+    >>> ints=behavior.create_intervals(activity_type = 'F', strain = 0
+                                       , mouse = 0, day = 0)
+    >>> print(sum(ints))
+
+    """
+
+    # Load intervals by activity type
+    l_ints = data.load_intervals(str(activity_type))
+
+    # Subset to M*2 array of  (start, stop) intervals based on
+    # specific mouse, strain and day
+    l_ints = l_ints[(l_ints.strain == strain) &
+                    (l_ints.mouse == mouse) &
+                    (l_ints.day == day)][['start', 'stop']]
+
+    # Create and return the intervals object
+    return(intervals.Intervals(l_ints))
```

Comment:
Remove the bracket around `return` here i.e. change:
` return(intervals.Intervals(l_ints))` =  `return intervals.Intervals(l_ints)`